ref/feature/transportSecurity/examples.adoc

```diff
@@ -0,0 +1,97 @@
+
+[source,java]
+----
+<ssl id="mySSLSettings"
+     keyStoreRef="myKeyStore"
+     trustStoreRef="myTrustStore"
+     clientAuthentication="true" />
+<keyStore id="LDAPKeyStore"
+          location="${server.config.dir}/myKey.p12"
+          type="PKCS12"
+          password="{xor}CDo9Hgw=" />
+<keyStore id="LDAPTrustStore"
+          location="${server.config.dir}/myTrust.p12"
+          type="PKCS12"
+          password="{xor}CDo9Hgw=" />
+----
+
+For more information, see link:/docs/ref/config/#ssl.html[SSL Repertoire].
+
+=== Multiple TLS configurations
+
+You can set multiple TLS configurations. If more than one TLS configuration is configured, then the default TLS configuration must be specified in the `server.xml` file that uses the `sslDefault` service configuration. The default configuration is specified by setting the `sslRef` attribute on the `sslDefault` element to equal the ID of the default configuration, as shown in the following example:
+
+
+[source,java]
+----
+<sslDefault sslRef="defaultSSLSettings" />
+<ssl id="defaultSSLSettings"
+     keyStoreRef="defaultKeyStore"
+     trustStoreRef="defaultTrustStore"
+     clientAuthenticationSupported="true" />
+<keyStore id="defaultKeyStore"
+          location="key.jks"
+          type="JKS" password="defaultPWD" />
+<keyStore id="defaultTrustStore"
+          location="trust.jks"
+          type="JKS" password="defaultPWD" />
+
+<ssl id="mySSLSettings"
+     keyStoreRef="myKeyStore"
+     trustStoreRef="myTrustStore"
+     clientAuthentication="true" />
+<keyStore id="LDAPKeyStore"
+          location="${server.config.dir}/myKey.p12"
+          type="PKCS12"
+          password="{xor}CDo9Hgw=" />
+<keyStore id="LDAPTrustStore"
+          location="${server.config.dir}/myTrust.p12"
+          type="PKCS12"
+          password="{xor}CDo9Hgw=" />
+----
+
+=== Java virtual machine (JVM) Default truststore configuration
+
+You can set the `trustDefaultCert` attribute to `true` to specify that the JVM default truststore can be used, in addition to the configured truststore, as shown in the following example:
+
+[source,java]
+----
+<ssl id="myDefaultSSLConfig"
+      keyStoreRef="defaultKeyStore"
+      trustStoreRef="defaultTrustStore"
+      trustDefaultCert="true" />
+----
+
+=== Outbound TLS configuration
+
+By default, Liberty uses the default TLS configuration for both inbound and outbound TLS connections. You can configure Liberty to have a global outbound default that allows you to configure one TLS configuration for the outbound default. Outbound TLS filters are configured on the `outboundConnection` element that is nested in the `ssl` element. You can specify either a host or a host and port to indicate where the outbound SSL connection is going by setting attributes on the `outboundConnection` element.
```

Comment:
"By default, Liberty uses the default TLS configuration for both inbound and outbound TLS connections." Can the 'by default mentioned in this sentence twice be reduced to once?
" You can specify either a host or a host and port to indicate where the outbound SSL connection is going by setting attributes on the `outboundConnection` element." In this sentence, instead of ' where the outbound SSL connection is going ' say ' where the outbound SSL connection goes'.